driver.sh

```diff
@@ -141,6 +141,11 @@ check_dependencies() {
   command -v timeout
   command -v unbuffer
 
+  for readelf in llvm-readelf-9 llvm-readelf-8 llvm-readelf-7 readelf; do
```

Comment:
I think this should have another `llvm-readelf` at the end (un-versioned, no GNU readelf)?

Reply:
Thanks for catching that, I don't know how that happens because I tested in both our Docker container and my server's regular environment :/